CHANGELOG.md

```diff
@@ -1,11 +1,24 @@
+## v0.7.0 [2022-04-22]
+
+_Enhancements_
+
+- Updated the data type of column `amount` in `digitalocean_bill` table to string. ([#46](https://github.com/turbot/steampipe-plugin-digitalocean/pull/46))
```

Comment:
@misraved I'd mark this as a bug fix, we should also mention what type we went from and went to, e.g., `Fixed the data type of column `amount` in `digitalocean_bill` table from int to string ...` (not sure if `int` is correct, but mainly for example purposes)